lnrpc/rpc.proto

```diff
@@ -1685,9 +1685,6 @@ message Invoice {
     */
     string memo = 1 [json_name = "memo"];
 
-    /// An optional cryptographic receipt of payment
-    bytes receipt = 2 [json_name = "receipt"];
```

Comment:
When removing proto fields, the field number should be marked `reserved`.

I also think we should just mark it deprecated for now, and remove it entirely in the future, along with other deprecated fields.

Reply:
Ok, rewinded the removal and just marked as deprecated.